backup/save/main.go

```diff
@@ -30,7 +30,7 @@ func main() {
 		panic(err)
 	}
 
-	p := provider.FileStoreType(help.GetEnv("CLOUD_PROVIDER", "LOCAL"))
+	p := provider.FileStoreType(help.GetEnv("CLOUD_PROVIDER", "FILE_SYSTEM"))
```

Comment:
what is file system?

Reply:
File system is the local filesystem. I can change it to LOCAL_FILESYSTEM instead of that clarified. Its just the value that the filestore enum uses

Reply:
Please do

Reply:
preferably set up a const even, best to not use strings like this when we mean them to be consts.

Reply:
There was a const setup, unsure why I didnt use it